conf/distro/include/omnect-os-distro.conf

```diff
@@ -73,28 +74,11 @@ WICVARS:append = " OMNECT_PART_SIZE_ETC"
 WICVARS:append = " OMNECT_PART_SIZE_DATA"
 WICVARS:append = " OMNECT_PART_OFFSET_UBOOT_ENV1"
 WICVARS:append = " OMNECT_PART_OFFSET_UBOOT_ENV2"
+WICVARS:append = " OMNECT_PART_OFFSET_BOOT"
 
 # on x86 poky sets these as default dependencies ... we don't use them
 WKS_FILE_DEPENDS_BOOTLOADERS:remove = "syslinux systemd-boot"
 
-# location of u-boot environment areas in KiB
-#   - Fixed addresses are used (--offset in WKS file), suitable for u-boot configuration.
-#   - It has to be aligned with the boot partition (OMNECT_PART_SIZE_BOOT) and the MBR/GPT.
-#   - Wic aborts the build, in the case the partitions cannot be located at the fixed location.
-#   - The partitions will be initialized with 0xff (--source in WKS file). In this way, u-boot is able
-#     to create the default environment during the first boot after flashing the SD-Card image.
-#   - No entries inside the partition table will be created (--no-table in WKS file).
-#   - The variable OMNECT_PART_SPARE1 is used to define a gap behind the boot partition.
-#   - The variable OMNECT_PART_SPARE2 is used to define a gap between both environment partitions.
-#     In this way, the environment partitions can be located in different erase blocks.
-OMNECT_PART_SPARE1 = "8192"
-OMNECT_PART_SPARE2 = "1984"
-OMNECT_PART_OFFSET_UBOOT_ENV1 ?= "${@int(d.getVar('OMNECT_PART_SIZE_BOOT')) + \
-                                    int(d.getVar('OMNECT_PART_SPARE1'))}"
-OMNECT_PART_OFFSET_UBOOT_ENV2 ?= "${@int(d.getVar('OMNECT_PART_OFFSET_UBOOT_ENV1')) + \
-                                    int(d.getVar('OMNECT_PART_SIZE_UBOOT_ENV')) + \
-                                    int(d.getVar('OMNECT_PART_SPARE2'))}"
-
```

Comment:
maybe it is worth keeping a general note about u-boot environment variables here w/o setting them, just to hint at their existence and purpose?
otoh, these should be later part of the planned glossary anyway.

Reply:
I take the missing reaction as vote for the glossary ;-)